lineman/app/models/membership_request_model.coffee

```diff
@@ -6,11 +6,12 @@ angular.module('loomioApp').factory 'MembershipRequestModel', (BaseModel) ->
 
     initialize: (data) ->
       @baseInitialize(data)
-      @fakeUser =
-        name: @name
-        email: @email
-        avatarKind: 'initials'
-        avatarInitials: _.map(@name.split(' '), (t) -> t[0]).join('')
+      if !@byExistingUser()
```

Comment:
it's not super pretty.. but it's how the model on the server is.

MembershipRequests are either by a real user or by a logged out user, which we track as name and email.

We need a better LoggedOutUser abstractiction that behaves and is used the same as a User